exercises/practice/resistor-color/.meta/example.gd

```diff
@@ -0,0 +1,16 @@
+const COLORS = ["black",
+                "brown",
+                "red",
+                "orange",
+                "yellow",
+                "green",
+                "blue",
+                "violet",
+                "grey",
+                "white"]
+
+func color_code(color):
```

Comment:
Small style guide note: in GDScript, methods are separated by 2 blank lines:
https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#blank-lines